inst/report_downloads/reportDocx.Rmd

```diff
@@ -190,12 +197,12 @@ if (!cm_ind) {
         )
       ) %>%
       dplyr::select(`Metric Name`, `Metric Description`, `Metric Value`, `Metric Score`) %>%
-      knitr::kable(format = 'pandoc')
+      kableExtra::kbl(format = 'pipe', booktabs = T)
   }
 }
 ```
 
-`r if ('Community Usage Metrics' %in% params$report_includes) {"* Metrics whose score is NA will not impact the package {riskmetric} score"}`
+`r if ('Community Usage Metrics' %in% params$report_includes) {"Metrics whose score is NA will not impact the package {riskmetric} score"}`
```

Comment:
Same as other footnote comment.

Reply:
The asterisk is currently missing from the column headers, so I will add it back in both the metric score header and the footnote.

Reply:
This looks more like it
![image](https://github.com/pharmaR/riskassessment/assets/75215958/b2fec596-40f6-447d-b287-450b4905b85c)